accelerometer/summariseEpoch.py

```diff
@@ -15,8 +16,8 @@ def getActivitySummary(epochFile, nonWearFile, summary,
     activityClassification=True, startTime=None, endTime=None,
     epochPeriod=30, stationaryStd=13, minNonWearDuration=60, mgMVPA=100,
     mgVPA=425, activityModel="activityModels/doherty2018.tar",
-    intensityDistribution=False, psd=False, fourierFrequency=False, m10l5=False, 
-    verbose=False, fourierWithAcc=False):
+    intensityDistribution=False, psd=False, fourierFrequency=False, fourierWithAcc=False, m10l5=False, 
```

Comment:
Can you make sure the calls to this method in accProcess.py (I don't think any other script calls it) are updated to reflect the updated order of parameters for this method?

Reply:
fixed in d79e6fa8f80be2016f778116baadc2c069ebfa86